src/main/java/FindFibonaci.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+
+public class FindFibonaci {
+
+    public String findFibonaci(int n){
+        String result = "";
+        ArrayList<Integer> arrResultInt = new ArrayList<>();
+        int i = 0;
+        if(n < 2)
+            return result + 0;
+        while (true){
+            if(i == 0 || i == 1){
```

Comment:
Cái lệnh 
```java
if(i == 0 || i == 1)
```

Anh thấy có thể thay bằng

```java
if(arrResultInt.size < 2)
```

Nhưng tốt hơn nữa, em hoàn toàn có thể thay lại

```java
arrResultInt.add(1);
arrResultInt.add(1);
do{

....
}while(..)
```

Như vậy, mình hoàn toàn có thể bỏ đi hẳn biến i